check-tarball.sh

```diff
@@ -12,7 +12,6 @@
 # was an issue we ran into.
 
 set -e
-set -x
```

Comment:
Mmmm.....

Reply:
The script already outputs some diagnostic information, going beyond that was too verbose for me to see what is going on.